utils/markdown.go

```diff
@@ -35,11 +35,19 @@ func StripMarkdown(markdown string) (string, error) {
 	return strings.TrimSpace(buf.String()), nil
 }
 
+var relLinkReg = regexp.MustCompile(`\[(.*)]\((/.*)\)`)
+var blockquoteReg = regexp.MustCompile(`^|\n(&gt;)`)
+
 // MarkdownToHTML takes a string containing Markdown and returns a string with HTML tagged version
-func MarkdownToHTML(markdown string) (string, error) {
+func MarkdownToHTML(markdown, siteURL string) (string, error) {
+	// Turn relative links into absolute links
+	absLinkMarkdown := relLinkReg.ReplaceAllFunc([]byte(markdown), func(s []byte) []byte {
```

Comment:
Let's use https://pkg.go.dev/regexp#Regexp.ReplaceAllStringFunc to avoid the double conversion.